### PR TITLE
added functionality to keep unsort the session dict

### PR DESCRIPTION
**Pull request by ritiksoni00**
#1668

### CODE REVIEW

File: tests/panels/test_request.py [hunk 1]
```diff
@@ -104,3 +104,22 @@ def test_namespaced_url(self):
...
+            "data": ["foo", "bar", 1],
+            (2, 3): "tuple_key",
+        }
+        data = {
+            "list": [(1, "value"), ("data", ["foo", "bar", 1]), ((2, 3), "tuple_key")]
+        }
+        response = self.panel.process_request(self.request)
+        self.panel.generate_stats(self.request, response)
+        panel_stats = self.panel.get_stats()
+        self.assertEqual(panel_stats["session"], data)
```
Comment (matthiask) [inline tests/panels/test_request.py:125]:
```suggestion
        self.assertEqual(panel_stats["session"], data)
        
        self.request.session = {
            "b": "b-value",
            "a": "a-value",
        }
        data = {
            "list": [("a", "a-value"), ("b", "b-value")]
        }
        response = self.panel.process_request(self.request)
        self.panel.generate_stats(self.request, response)
        panel_stats = self.panel.get_stats()
        self.assertEqual(panel_stats["session"], data)        
```

**Review by matthiask — COMMENTED** [review]
Looks good! I think you're only verifying the failing case though, maybe add a second test where the session contains only string keys (which can actually be sorted)?

File: tests/panels/test_request.py (hunk 1 above)
Comment (ritiksoni00) [reply tests/panels/test_request.py:125]: i guess I should also test that in a new test

Comment (matthiask) [reply tests/panels/test_request.py:125]: I think it's fine to keep both in a single test (as suggested)

Comment (tim-schilling) [reply tests/panels/test_request.py:125]: @ritiksoni00 you can either copy this code into your local environment and commit or use github's "Commit suggestion" button to create a commit on the PR. When we merge it, we'll squash the commits into one.

**Review by tim-schilling — CHANGES_REQUESTED** [review]
Thank you for opening a PR! Please add a mention of this change in `docs/changes.rst`. That helps reduce effort in releasing new versions.

**Review by tim-schilling — APPROVED** [review]

**Comment by tim-schilling** [comment]
Thank you for the work on this @ritiksoni00!

**Comment by ritiksoni00** [comment]
> Thank you for opening a PR! Please add a mention of this change in `docs/changes.rst`. That helps reduce effort in releasing new versions.

sorry, tim for not responding.. !

**Comment by tim-schilling** [comment]
No worries, thank you for the PR!

**Pull request merged**